src/Sentry/Internal/SerializableExtensions.cs

```diff
@@ -4,18 +4,18 @@
 using System.Threading.Tasks;
 using Sentry.Protocol.Envelopes;
 
-namespace Sentry.Tests.Helpers
+namespace Sentry.Internal
 {
     internal static class SerializableExtensions
     {
         public static async Task<string> SerializeToStringAsync(
             this ISerializable serializable,
             CancellationToken cancellationToken = default)
         {
-#if !NET461 && !NETCOREAPP2_1
+#if !NET461 && !NETSTANDARD2_0
             await
 #endif
-            using var stream = new MemoryStream();
+                using var stream = new MemoryStream();
```

Comment:
```suggestion
#if !NET461 && !NETSTANDARD2_0
            await using var stream = new MemoryStream();
#else
            using var stream = new MemoryStream();
#endif
```

Reply:
Why is it beneficial to add a `#else` block with the same statement over only adding `await`?

If we agreed on this new style lets do a single pass and replace it everything. The approach I used is already the standard and there are many usages